css/hnn.less

```diff
@@ -0,0 +1,335 @@
+@import "colors";
+
+html,
+body {
+  height: 100%;
+  background: white !important;
+}
+
+#footer {
+  z-index: 2;
+}
+
+#mainContainer {
+  background-size: 100% 100%;
+  overflow: auto;
+  height: 100%;
+}
+#mainContainer > div {
+  height: 100% !important;
+}
+.actionButton {
```

Comment:
are we using these classes or do we know we will use them?

Reply:
I don't think I can answer that question, but I can tell you that I agree that is good practice to not add stuff that we think we might need in the future, so I removed everything besides the bare minimum, and will be adding the rest if/when needed.